lib/hooks/api/useContributorList.ts

```diff
@@ -0,0 +1,52 @@
+import useSWR, { Fetcher } from "swr";
+import { useState } from "react";
+import publicApiFetcher from "lib/utils/public-api-fetcher";
+
+export function convertToContributors(rawContributors: DBListContributor[] = []): DbPRContributor[] {
+  const contributors = rawContributors
+    ? rawContributors.map((contributor) => {
+        return {
+          author_login: contributor.login,
+          updated_at: contributor.created_at,
+          user_id: contributor.id,
+        };
+      })
+    : [];
+
+  return contributors;
+}
+
+export const useContributorsList = ({
+  listId,
+  initialData,
+  initialPage = 1,
+  defaultLimit = 10,
+}: {
+  listId: string | undefined;
+  initialData?: {
+    data: DbPRContributor[];
+    meta: Meta;
+  };
+  initialPage?: number;
+  defaultLimit?: number;
+}) => {
+  const [page, setPage] = useState(initialPage);
+  const [limit, setLimit] = useState(defaultLimit);
+  const { data, error, mutate } = useSWR<any>(
```

Comment:
nit: why `any`?

Reply:
I think I might have copy pasted that. Will update.